common/presenters/moves-by-vehicle.js

```diff
@@ -0,0 +1,77 @@
+const { addMinutes, formatISO, isToday, parseISO } = require('date-fns')
+const { isEmpty, get, groupBy, orderBy, sortBy } = require('lodash')
+
+const setMissingToLocation = require('../helpers/move/set-missing-to-location')
+
+const moveToCardComponent = require('./move-to-card-component')
+
+module.exports = function movesByVehicle(data, locationKey = 'to_location') {
+  const locations = []
+  const groupedByLocation = groupBy(data, 'meta.vehicle_registration')
+
+  Object.entries(groupedByLocation).forEach(([group, moves], index) => {
+    const vehicleRegistration = get(moves[0], 'meta.vehicle_registration')
+    const expectedTimeMetaKey =
+      locationKey === 'to_location'
+        ? 'expected_collection_time'
+        : 'expected_time_of_arrival'
+
+    // TODO: Remove dummy times below
```

Comment:
Do these sample times need to be removed before this PR can be merged?

Reply:
Yep :) I left the `TODO` in there to pick that up.

You can ignore that block as part of the initial review. This PR is mostly still draft but I wanted to get early eyes in case there was any glaring things that need addressing.